readme.md

```diff
@@ -19,7 +21,7 @@ git clone https://github.com/nextcloud/nextcloud.com.git
 - Build the stack
 
 ```
-docker-compose up -d
+sudo docker-compose up -d
```

Comment:
This is advised against, we should add that you should add your user to the docker group... Let me see the docs